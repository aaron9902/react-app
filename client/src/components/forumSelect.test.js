import React from 'react';
import { mount } from 'enzyme';
import ForumSelect from './forumSelect';
import { BrowserRouter, BrowserRouter as Router, Link } from "react-router-dom"; 

describe('<ForumSelect/ >', () => {

  
//StepID: S401
  it('matches snapshot', () => {
    const wrapper = mount(<BrowserRouter><ForumSelect /></BrowserRouter>);
    expect(wrapper).toMatchSnapshot();
  });

});
