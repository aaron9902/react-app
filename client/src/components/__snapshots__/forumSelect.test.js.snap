// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ForumSelect/ > matches snapshot 1`] = `ReactWrapper {}`;
